Components/Typography/typography.css

```diff
@@ -0,0 +1,44 @@
+.typography-box {
+  border: 1px solid var(--gray);
+  margin: auto;
+  padding: 1.5rem;
+  border-radius: 6px;
+  margin-bottom: 1.5rem;
+}
+.heading-1,
+.heading-2,
+.heading-3,
+.heading-4,
+.heading-5,
+.heading-6 {
+  margin: 1rem;
+}
```

Comment:
you can have a utility class for margin, just a suggestion you can ignore if you want.